README.md

```diff
@@ -78,6 +82,12 @@ be any production-ready Pod providers. Only the one prototypes in multiple place
 - [solid.community](https://solid.community/) - Hosted by (?)) on (?)).
 - [Solid Web](https://solidweb.org/) - Hosted by Matthias Evering on (?)
 
+## Servers
+
+- [PHP Solid Server](https://github.com/pdsinterop/solid-php-server)
+  \- Solid Server written in PHP
+  ([repo](https://github.com/pdsinterop/solid-php-server))
```

Comment:
Repo link is maybe superfluous if it's the same as the main project link?

Reply:
Good catch! My bad :disappointed_relieved: , should be https://pdsinterop.org/php-solid-server/. Will fix.